components/library/glib/glib2.p5m

```diff
@@ -361,23 +347,23 @@ file path=usr/lib/$(MACH64)/pkgconfig/gmodule-export-2.0.pc
 file path=usr/lib/$(MACH64)/pkgconfig/gmodule-no-export-2.0.pc
 file path=usr/lib/$(MACH64)/pkgconfig/gobject-2.0.pc
 file path=usr/lib/$(MACH64)/pkgconfig/gthread-2.0.pc
-file usr/lib/gio-launch-desktop path=usr/bin/gio-launch-desktop mode=0555
+file path=usr/lib/gio-launch-desktop
 file path=usr/lib/glib-2.0/include/glibconfig.h
```

Comment:
This
file usr/lib/gio-launch-desktop path=usr/bin/gio-launch-desktop mode=0555
is essential to be able to start apps with the mouse.